svgelements/svgelements.py

```diff
@@ -43,7 +43,7 @@
 and the Arc can do exact arc calculations if scipy is installed.
 """
 
-SVGELEMENTS_VERSION = "1.6.8"
+SVGELEMENTS_VERSION = "1.6.9"
```

Comment:
There's also a version reference in setup.py or whatever.

Reply:
Done.